2996_reflection/reflection.md

```diff
@@ -5676,16 +5669,16 @@ Extend and re-format paragraph 3 of [temp.names]{.sref}:
 
 ::: std
 
-[3]{.pnum} A `<` is interpreted as the delimiter of a `$template-argument-list$` if it follows
+[3]{.pnum} A `<` is interpreted as the delimiter of a `$template-argument-list$` if
 
-* [[#.#]{.pnum} a `$splice-specifier$` that either appears in a type-only context or is preceded by `template` or `typename`, or]{.addu}
-* [#.#]{.pnum} a name that is not a `$conversion-function-id$` and
-  * [#.#.#]{.pnum} that follows the keyword template or a ~ after a nested-name-specifier or in a class member access expression, or
-  * [#.#.#]{.pnum}  for which name lookup finds the injected-class-name of a class template or finds any declaration of a template, or
+* [[#.#]{.pnum} it follows a `$splice-specifier$` that either appears in a type-only context or is preceded by the keyword `template` or the keyword `typename`, or]{.addu}
```

Comment:
make this two bullets, it's unclear what the "or" applies to.

> * it follows a splice-specifier that either
>   * appears in a type-only context or
>   * is preceded by the keyword `template` or the keyword `typename`

Reply:
Done